ibm/service/vpc/resource_ibm_is_vpc.go

```diff
@@ -112,7 +134,173 @@ func ResourceIBMISVPC() *schema.Resource {
+							Optional:    true,
+							Computed:    true,
+							Description: "Indicates whether this VPC is enabled as a DNS name resolution hub.",
+						},
+						isVPCDnsResolutionBindingCount: &schema.Schema{
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: "The number of DNS resolution bindings for this VPC.",
+						},
+						isVPCDnsResolver: &schema.Schema{
+							Type:        schema.TypeList,
+							MinItems:    1,
+							MaxItems:    1,
+							Optional:    true,
+							Computed:    true,
+							Description: "The DNS resolver configuration for the VPC.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									isVPCDnsResolverServers: &schema.Schema{
+										Type:        schema.TypeList,
+										Computed:    true,
+										Description: "The DNS servers for this VPC. The servers are populated:- by the system when `dns.resolver.type` is `system`- using the DNS servers in `dns.resolver.vpc` when `dns.resolver.type` is `delegated`- using `dns.resolver.manual_servers` when the `dns.resolver.type` is `manual`.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												isVPCDnsResolverManualServersAddress: &schema.Schema{
+													Type:        schema.TypeString,
+													Computed:    true,
+													Description: "The IP address.This property may add support for IPv6 addresses in the future. When processing a value in this property, verify that the address is in an expected format. If it is not, log an error. Optionally halt processing and surface the error, or bypass the resource on which the unexpected IP address format was encountered.",
+												},
+												isVPCDnsResolverManualServersZoneAffinity: &schema.Schema{
+													Type:        schema.TypeString,
+													Computed:    true,
+													Description: "Zone name, if present, DHCP configuration for this zone will have this DNS server listed first.",
+												},
+											},
+										},
+									},
+									isVPCDnsResolverType: &schema.Schema{
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: "The type of the DNS resolver used for the VPC.- `delegated`: DNS server addresses are provided by the DNS resolver of the VPC               specified in `dns.resolver.vpc`.- `manual`: DNS server addresses are specified in `dns.resolver.manual_servers`.- `system`: DNS server addresses are provided by the system.",
+									},
+									isVPCDnsResolverVpc: &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The VPC whose DNS resolver provides the DNS server addresses for this VPC.The VPC may be remote and therefore may not be directly retrievable.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												isVPCDnsResolverVpcCrn: &schema.Schema{
```

Comment:
isVPCDnsResolverVpcCrn and isVPCDnsResolverVpcId. should be marked Optional as well
and should be mutually exclusive.

Reply:
addressed